react/src/components/nplusone.js

```diff
@@ -0,0 +1,53 @@
+import { Component } from "react";
+import Context from "../utils/context";
+import { Link, useFetcher } from "react-router-dom";
+import * as Sentry from "@sentry/react";
+import { connect } from "react-redux";
+import { determineBackendUrl } from "../utils/backendrouter";
+class NPlusOneItem extends Component {
+  async componentDidMount() {
+    console.log("Mounting NPlusOneItem");
+    const response = await fetch(this.props.backend + "/product/0/info?id=" + this.props.index);
+    console.log({ response });
+    const json = await response.json();
+    console.log({ json });
+    this.setState({
+      data: json,
+    });
+  }
+  render() {
+    return <li>{this.props.index}</li>;
+  }
+}
+class Nplusone extends Component {
+  constructor(props) {
+    super(props);
+    const ids = new Array(20).fill(0).map((item, index) => index);
+    this.state = {
+      ids,
+    };
+  }
+  async componentDidMount() {
+    console.log("Mounting NPlusOne with props", this.props);
```

Comment:
Hey could you remove all the other console.log's you committed? I'm okay with this one in componentDidMount, just so we have a log that the component is properly mounting.